arm/Microsoft.Network/connections/version.json

```diff
@@ -1,4 +1,4 @@
 {
     "$schema": "https://raw.githubusercontent.com/dotnet/Nerdbank.GitVersioning/master/src/NerdBank.GitVersioning/version.schema.json",
-    "version": "0.4"
+    "version": "1.0"
```

Comment:
I understand this is a breaking change so major is to be updated. Are we OK to publish 1.0 modules before GA?